aiida/repository/repository.py

```diff
@@ -32,7 +32,7 @@ class keeps a reference of the virtual file hierarchy. This means that through t
 
     # pylint: disable=too-many-public-methods
 
-    _backend = None
+    _backend: Optional[AbstractRepositoryBackend] = None
```

Comment:
why does this need to be a class attribute and not a regular one?

Reply:
In principle it doesn't, we initialize it in the constructor anyway, so I think we can get rid of it.

Reply:
removed